src/service/http/nebula/web.js

```diff
@@ -553,6 +553,9 @@ const onQueryResult = (state, r) => {
     else if (state.metrics.length == 1 && state.metrics[0].M == neb.Rollup['TREEMERGE']) {
         choices(['icicle', 'flame']);
     }
+    else if (state.metrics.length == 1 && state.metrics[0].M == neb.Rollup['HIST']) {
```

Comment:
I think you can move the data transformation logic from charts2.js to here and update `ds`. The benefit is
- chart lib stays generic
- later we will display parsed result in table rather than the single json blob.